ofl/notonaskharabic/METADATA.pb

```diff
@@ -10,15 +10,21 @@ fonts {
   filename: "NotoNaskhArabic[wght].ttf"
   post_script_name: "NotoNaskhArabic-Regular"
   full_name: "Noto Naskh Arabic Regular"
-  copyright: "Copyright 2019 Google LLC. All Rights Reserved."
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/arabic)"
 }
 subsets: "arabic"
+subsets: "latin"
+subsets: "latin-ext"
 subsets: "menu"
 axes {
   tag: "wght"
   min_value: 400.0
   max_value: 700.0
 }
+source {
+  repository_url: "https://github.com//notofonts/arabic.git"
+  archive_url: "https://github.com/notofonts/arabic/releases/download/NotoNaskhArabic-v2.013/NotoNaskhArabic-v2.013.zip"
```

Comment:
Title of this PR states v2.014, does this need to be updated?

Reply:
Discussed during the pipeline meeting today.  It sounds like it does require an update.